### DISCUSSION TITLE
Addition of "Effect Timers"

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15204] [Ideas]
A certain other client has a plugin called "Effect Timers" which showed a small icon on NPCs and your own player to show when an effect like vengeance, imbued heart of freeze was applied. This is very helpful in various PVE scenarios such as thermy and mutadile to show when freeze will wear off, and to show your team mates venge status.

Would be a great quality of life addition to the game even without any pvp features such as showing teleblock timers

![image](https://user-images.githubusercontent.com/11243032/175552357-d53e74b3-9678-4d33-9911-83b6a42df8a3.png)

**Comment by Adam-** [comment]
It doesn't really matter what "other clients" are doing if it is against ToS, we basically can't add it/or are unwilling to add it. See https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1

**Reply by ghost** [reply]
Thanks for that link - Not sure why I couldnt find it in a search.
My mistake
